File: android/src/main/java/com/hnguyen48206/blesrv/BLEServ.java
package com.hnguyen48206.blesrv;

import android.util.Log;

public class BLEServ {


    public String startService(String value) {
        Log.i("Echo", value);
        return value;
    }

    public String stopService(String value) {
        Log.i("Echo", value);
        return value;
    }
}

File: ios/Sources/BLEServPlugin/BLEServ.swift
import Foundation

@objc public class BLEServ: NSObject {
    @objc public func echo(_ value: String) -> String {
        print(value)
        return value
    }
     @objc public func startService(_ value: Any) -> Any {
        print(value)
        return value
    }
     @objc public func stopService(_ value: Any) -> Any {
        print(value)
        return value
    }
}
